### PR TITLE
spec: rename python binary packages

### PR DESCRIPTION
**Pull request by mkutlak**
Python 2 binary package renamed to python2-abrt
See https://fedoraproject.org/wiki/FinalizingFedoraSwitchtoPython3

Python 3 binary package renamed to python3-abrt
Add-on packages renamed to python[23]-abrt-addon
Documentation packages renamed to python[23]-abrt-doc

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

**Comment by mkutlak** [comment]
retest this please

File: abrt.spec.in [hunk 1]
```diff
@@ -281,25 +281,31 @@ Provides: libreport-plugin-bodhi = %{version}-%{release}
...
 Summary: %{name}'s addon for catching and analyzing Python exceptions
-Requires: python
 Requires: %{name} = %{version}-%{release}
 Requires: systemd-python
-Requires: abrt-python
+Requires: python2-abrt
+# Remove before F30
+Provides: abrt-addon-python = %{version}-%{release}
+Provides: abrt-addon-python%{?_isa} = %{version}-%{release}
+Obsoletes: abrt-addon-python < %{version}-%{release}
```
Comment (xsuchy) [inline abrt.spec.in]: Here should be last version-release+1 of abrt-addon-python
See https://fedoraproject.org/wiki/Packaging:Guidelines#Renaming.2FReplacing_Existing_Packages

File: abrt.spec.in [hunk 2]
```diff
@@ -281,25 +281,31 @@ Provides: libreport-plugin-bodhi = %{version}-%{release}
...
 
-%package addon-python
+%package -n python2-abrt-addon
 Summary: %{name}'s addon for catching and analyzing Python exceptions
-Requires: python
 Requires: %{name} = %{version}-%{release}
 Requires: systemd-python
-Requires: abrt-python
+Requires: python2-abrt
+# Remove before F30
```
Comment (xsuchy) [inline abrt.spec.in]: This is really not needed when provides/obsoletes are set correctly

Comment (mkutlak) [reply abrt.spec.in]: Packages should use python2- version if it is available.

https://fedoraproject.org/wiki/Packaging:Python#Dependencies

**Comment by mkutlak** [comment]
Changed the variable version to static one.

**Pull request merged**